src/controllers/trust.interrupt.controller.ts

```diff
@@ -40,10 +42,13 @@ const post = (
     logger.debugRequest(req, `${req.method} ${req.route.path}`);
 
     const firstTrustId = "1";
-    return safeRedirect(res, `${config.TRUST_ENTRY_URL + "/" + firstTrustId}`);
+    let trustEntryUrl = config.TRUST_ENTRY_URL;
+    if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL) && req) {
```

Comment:
you don't need the && req check here, it should always be present in this post function. We only added it into a previous one because the req was an optional parameter so we had to check it existed before using it.

Reply:
Removed.